source/guide_neos.rst

```diff
@@ -0,0 +1,155 @@
+.. highlight:: console
+
+.. sidebar:: Logo
+  
+  .. image:: _static/images/neos.png 
+      :align: center
+
+#########
+Neos
+#########
+
+Neos_ is an open source Content Application Platform based on its own PHP framework Flow and distributed under the GPLv3 licence.
```

Comment:
The introduction seems a little short to me. What does it do? What can I use it for? Why it is great? Also try to include any words people might search for in the text. Thanks :)